crowd-server/crowd_server/apps/food_fact/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from .models import Question,Answer
+
+
+class QuestionSerializer(serializers.ModelSerializer):
+
+    class Meta:
+    
+        model = Question
+        fields = ['language','img_url' ,'question','response_count']       
+
+class ResponseSerializer(serializers.ModelSerializer):
```

Comment:
Rename ResponseSerializer to AnswerSerializer

Reply:
done